doc/changes/DM-32968.misc.rst

```diff
@@ -0,0 +1 @@
+Make HTCondor handle all jobs exiting with a signal uniformly.
```

Comment:
Probably should change description a little bit so that users know what it means.  Something like "Make HTCondor treat all jobs exiting with a signal as if they ran out of memory."